src/API/Recruit.Api/Mappers/TraineeshipVacancyMapper.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Linq;
+using Esfa.Recruit.Vacancies.Client.Domain.Entities;
+using SFA.DAS.Recruit.Api.Models;
+using ApplicationMethod = Esfa.Recruit.Vacancies.Client.Domain.Entities.ApplicationMethod;
+using DurationUnit = Esfa.Recruit.Vacancies.Client.Domain.Entities.DurationUnit;
+using EmployerNameOption = Esfa.Recruit.Vacancies.Client.Domain.Entities.EmployerNameOption;
+using QualificationWeighting = Esfa.Recruit.Vacancies.Client.Domain.Entities.QualificationWeighting;
+using WageType = Esfa.Recruit.Vacancies.Client.Domain.Entities.WageType;
+
+namespace SFA.DAS.Recruit.Api.Mappers
+{
+    public static class TraineeshipVacancyMapper
+    {
+        public static Vacancy MapFromCreateTraineeshipVacancyRequest(this CreateTraineeshipVacancyRequest request, Guid id, VacancyType vacancyType)
+        {
+            return new Vacancy
+            {
+                EmployerLocation = new Address
+                {
+                    AddressLine1 = request.Address.AddressLine1,
+                    AddressLine2 = request.Address.AddressLine2,
+                    AddressLine3 = request.Address.AddressLine3,
+                    AddressLine4 = request.Address.AddressLine4,
+                    Postcode = request.Address.Postcode
+                },
+                Id = id,
+                Description = request.Description,
+                Title = request.Title,
+                EmployerName = request.EmployerName,
+                LegalEntityName = request.LegalEntityName,
+                ApplicationMethod = (ApplicationMethod?)CreateTraineeshipVacancyApplicationMethod.ThroughFindATraineeship,
```

Comment:
This shouldn't need setting - it should default to this on the type being Traineeship